README.md

```diff
@@ -5,7 +5,7 @@ Make it easier for developers to use [JSONForms](https://github.com/eclipsesourc
 ## Continuous Integration
 The JSONForms project is build and tested via [Travis](https://travis-ci.org/). Coverage is documented by [Coveralls](https://coveralls.io).
 
-Current status: [![Build Status](https://travis-ci.org/eclipsesource/jsonforms-tooling.svg?branch=master)](https://travis-ci.org/eclipsesource/jsonforms-tooling) [![Coverage Status](https://coveralls.io/repos/eclipsesource/jsonforms-tooling/badge.svg?branch=master&service=github)](https://coveralls.io/github/eclipsesource/jsonforms-tooling?branch=master)
```

Comment:
Why did you change the URLs?

Reply:
Actually a good question. I undid this change